components/suggest/src/schema.rs

```diff
@@ -113,9 +113,11 @@ CREATE TABLE yelp_location_signs(
 ) WITHOUT ROWID;
 
 CREATE TABLE yelp_custom_details(
-    icon_id TEXT PRIMARY KEY,
+    icon_light_theme_id TEXT NOT NULL,
+    icon_dark_theme_id TEXT NOT NULL,
```

Comment:
Storing both light and dark icons in the same row is OK with me, and also I just wanted to mention another way to do it would be to have one icon per row plus an "icon type" column. If we need to support any more types of icons in the future -- and I don't know why we would! -- I think we should switch to that approach.

Reply:
Yeah, if we do not know how many icons we should handle, then yeah, should introduce the mechanism.